proto/cosmos/evidence/module/v1/module.proto

```diff
@@ -7,6 +7,6 @@ import "cosmos/app/v1alpha1/module.proto";
 // Module is the config object of the evidence module.
 message Module {
   option (cosmos.app.v1alpha1.module) = {
-    go_import: "cosmossdk.io/x/evidence"
+    go_import: "github.com/cosmos/cosmos-sdk/x/evidence"
```

Comment:
You still need this, but you need to add a replace for API as well, when you re-generate the protos.

Reply:
+1, I had the same confusion
this needs more clarity, leaving this as of now

Reply:
Updating `protocgen.sh` would let you have the correct name:

```shell
# move proto files to the right places
cp -r github.com/cosmos/cosmos-sdk/* ./
cp -r cosmossdk.io/** ./
rm -rf github.com
rm -rf cosmossdk.io
```

Reply:
wdyt we should do here julien ? should I rename the proto files ?

Reply:
I think we should add the above snippet to `protocgen.sh` (https://github.com/cosmos/cosmos-sdk/blob/julien/rename-nft/scripts/protocgen.sh#L31-L35) and do modify the `go_import`. As the package is renamed, so the `go_import` should reflect that.